### PR TITLE
Leaderboard Setup

### PR DESCRIPTION
**Pull request by roodjinocherilus**
In this Pull Request, I have set up the project for the Leaderboard list app, using webpack and ES6 features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,10 +33,10 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Awesome books] <a name="about-project"></a>
+# 📖 [LeaderBoard] <a name="about-project"></a>
 
 
-**WebPack and Linters Template** This is a template Repository for webpack project with Linters pre-installed
+**LeaderBoard** The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
```
Comment (rloterh) [inline README.md:40]: - [x] Live demo link one _line 64_ is broken, kindly fix it!

Comment (rloterh) [inline README.md:40]: - [ ] [OPTIONAL] Kindly endeavor to remove the guide note below the MIT license, on _line 186_.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Roodjino, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦   🏃

## Highlights 👍  
- Project is properly configured!
- App runs correctly!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (roodjinocherilus) [reply README.md:40]: I did not add any link! I just said, it is pending. Since the project is not complete yet, I have not deployed it

Comment (rloterh) [reply README.md:40]: @roodjinocherilus 
We ought to remove the link in markdown, as it remained clickable in the ReadMe preview.

A visitor to this repo will most likely be unaware of your intentions in the markdown!

Cheers! 😃

Comment (rloterh) [reply README.md:40]: Needs attention!

Comment (rloterh) [inline README.md:40]: - [ ] Live demo link in your ReadMe on _line 64_ is broken, kindly endeavor to fix or remove it if this app is not yet deployed.

**Clicking on the Live demo link in your ReadMe..**
image

**Leads to a 404 request error page:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Roodjino, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🏃 

## Highlights 👍  
- Project is set up correctly!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (roodjinocherilus) [reply README.md:40]: Okay @rloterh Thank you for the feedback. Just done it

**Review by BenMukebo — APPROVED** [review]
### Hi @roodjinocherilus, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Gitflow was correctly used. 👍🏻 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**